src/Thumbnail/Schema/Thumbnail.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Thumbnail\Schema;
+
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
+/**
+ * @internal
+ */
+#[Schema(
+    title: 'Thumbnail',
+    required: ['id', 'text'],
+    type: 'object'
+)]
+final readonly class Thumbnail
```

Comment:
Please add the additional attributes here and the thumbnail event :)

Reply:
I was just waiting till the Auth will be merged and forgot to convert this PR to draft, but I will do the changes now :)